js/lang/fr.js

```diff
@@ -7,40 +7,40 @@ app.lang['fr'] = (function() {
 		'language.cs': 'Czech',
 		'language.en': 'English',
 		'language.es': 'Spanish',
-		'language.fr': 'Frrançais',
-		'main.message.status.0': 'Erreur: y a pas de reseau',
+		'language.fr': 'Français',
+		'main.message.status.0': 'Erreur: il n\'y a pas de réseau',
 		'menu.new-payment': 'Nouveau Paiement',
 		'menu.settings': 'Parametres',
 		'menu.payment-history': 'Histoire des paiements',
-		'settings.title': 'Parametres titre',
```

Comment:
Paramètres, sans __titre__

Reply:
peut etre Parametres avec titre je pense

Reply:
Je ne suis pas d'accord. Dans js/lang/en.js, je lis :
```
'settings.title': 'Settings',
```
et "Settings" se traduit par "Paramètres" en français.

Reply:
what about the title?

Reply:
What do you mean? If you're thinking that you need to translate "settings.title" by "paramètres.titre", then you have not really understood the task. But since you translated "menu.settings" by "Paramètres" and not "menu.paramètres", I assumed you knew that the task was about translating the text in js/lang/en.js into French -- by which I mean translating the value, and not the key, of the JavaScript object. Does that make sense?

Reply:
why then the pay-choose-method.cancel': 'Cancel', is translated as cancel in es.js version if the key is not title in 'settings.title':

Reply:
It is translated as "Cancelar" in es.js.
Anyway, in JavaScript, an object is written in curly braces and consists of key-value pairs, like so
```
var myObject = {
    "key1": "value1",
    "key2": "value2"
}
```
So here, it is not so much about translating the key BUT translating the value of the key into French. So when you see "Cancel" or "Cancelar" in English or Spanish for the key "pay-choose-method.cancel", then you must translate it by "Annuler" in French. As simple as this. Is that clear now?

Reply:
i know what is an object in js. what i am saying is that setting.title is referencing to the parametres  titre in french. I do not speak spanish

Reply:
I speak French, and the translation of "Settings" into French is "Paramètres".  So if in `en.js`, we have
```
{
...
"settings.title": "Settings",
...
}
```
then in `fr.js`, we should have
```
{
...
"settings.title": "Paramètres",
...
}
```
Does that make sense?

Reply:
I speak both french and english, i miss the accents because my keyboard is english. and fyi it does not make sense the way the key has been made. if the title should not translated the key should be then title.settings rather than settings.title. things should be consistent

Reply:
I was trying to be cordial and beg your pardon if my prose appears otherwise. It is expected the translation will respect the spelling in the target language, so translating  into a language requiring diacritics such as acute accents, grave accents etc. means these should be present. 
If you believe the translation keys should be organised differently, please share your ideas and make a PR to update them and we can discuss that there and then. But the task at hand is to translate English text into French, not whether the keys make sense or not.